src/platform/site-wide/sass/shame.scss

```diff
@@ -571,3 +571,27 @@ img[data-srcset] {
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
 }
+
+.no-p-bottom-margin p {
+  margin-bottom: 0;
+}
+
+.operating-status-title {
+  margin: auto;
+}
+
+.operating-status-info {
+  min-height: 70px;
+  padding: 10px 0;
+  .normal-hours {
```

Comment:
if possible we should avoid nesting queries. `normal-hours` looks unique to this page. also there are design system utilities for that will probably work: 
https://design.va.gov/utilities/display
https://design.va.gov/utilities/margins#margin-top

Reply:
:+1: @rianfowler